core/src/main/java/org/infinispan/notifications/cachemanagerlistener/CacheManagerNotifierImpl.java

```diff
@@ -133,7 +135,10 @@ private void invokeListener(ListenerInvocation listener, EventImpl e) {
       } catch (Exception x) {
          // Only cache entry-related listeners should be able to throw an exception to veto the operation.
          // Just log the exception thrown by the invoker, it should contain all the relevant information.
-         log.error(x);
```

Comment:
Alright, I just didn't know cause logging an entire stack trace to error can spam the output quite a bit possibly (even with no trace configuration), but I am open to either way.